package.json

```
@@ -24,7 +24,7 @@
     "pretest": "npm run lint:frontend",
     "test": "jest app/javascript/ bin/ --coverage",
     "test:watch": "jest app/javascript/ bin/ --watch",
-    "postcss": "postcss public/assets/*.css -d public/assets"
+    "postcss": "postcss public/assets/*.css -d public/assets 2> err.log"
```

question (non-blockin): Is `2>` the same as `&>`? For context, see https://github.com/forem/forem/issues/11490#issuecomment-731256928

---

@nickytonline `2>` only redirects standard error, `&>` redirects both standard output and standard error to the log file, it's a shortcut for `2>&1` I think

---

Thanks for the explanation @rhymes.

---

Based on the answer [here](https://stackoverflow.com/questions/24793069/what-does-do-in-bash), `&>file` would log everything into the said file, which is exactly identical to  `1>file 2>file` (1 for standard output, 2 for standard error). I omitted the `1>` because I only want to store the error.

Now that I said that, I think there's a benefit to storing both together. let me update it.

```suggestion
    "postcss": "postcss public/assets/*.css -d public/assets &> precompile-postcss.log"
```